notifier.go

```diff
@@ -44,14 +44,18 @@ func colorizeUpdateMessage(updateString string) (string, error) {
 	maxContentWidth := line1Length
 	line1Rest := maxContentWidth - line1Length
 
-	terminal, err := ts.GetSize()
-	if err != nil {
-		return "", err
+	// termWidth is set to a default one to use when we are
+	// not able to calculate terminal width via OS syscalls
+	termWidth := 25
+
+	terminalWidth, err := pb.GetTerminalWidth()
```

Comment:
if width, err := pb.GetTerminalWidth(); err == nil {
    termWidth = width
}

Use shorter names for temporarily scoped variables. Longer names are often more meaningful and indicates that they are scoped to live longer.